#! /usr/bin/python3


try:

	while(1)::
		print("Infinite Loop")
	
except:
	print("Exception caught")
